src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java

```diff
@@ -69,7 +69,7 @@ public JsonObject asGeoJsonFeature(final String flagIdentifier)
         final JsonObject properties;
         if (locationItem != null)
         {
-            feature = locationItem.asGeoJsonFeature();
+            feature = locationItem.asGeoJsonGeometry();
```

Comment:
is this a breaking change?

Reply:
Yeah, on the Atlas side (5.3.1)

Reply:
ahh I hate breaking changes... @matthieun  what's the reason to rename this method?